app/models/domain_cron.rb

```diff
@@ -78,4 +78,46 @@ def self.start_redemption_grace_period
     STDOUT << "#{Time.zone.now.utc} - Successfully set server_hold to #{marked} of #{real} domains\n" unless Rails.env.test?
     marked
   end
+
+  def self.start_client_hold
+    log_prepare_client_hold
+
+    ::PaperTrail.whodunnit = "cron - #{__method__}"
+
+    marked = 0
+    real = 0
+
+    Domain.force_delete_scheduled.each do |domain|
+      next unless domain.client_holdable?
+
+      real += 1
+      domain.statuses << DomainStatus::CLIENT_HOLD
+      log_start_client_hold(domain)
+
+      domain.save(validate: false) and marked += 1
```

Comment:
Maybe use `&&` ?

Reply:
Well, that's some semantical && ideological point here.
You see, operators 'and' and 'or' are low precedence flow control operators useful to chain methods, not boolean operators like '&&' and '||'.
See http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/ for some examples.

Reply:
Oh, right, missed the real intention of it. Thanks for clarification